src/App.js

```diff
@@ -1,25 +1,69 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React from 'react';
+import axios from 'axios';
+
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchQuery: '',
+      cityData: {}
+    }
+  }
+
+  handleSubmit = e => {
+    e.preventDefault();
+    let city = e.target.city.value
+    this.setState({
+      searchQuery: e.target.city.value,
+    });
+    this.getCityInfo(city);
+  }
+  
+  getCityInfo = async (city) => {
+    try {let url = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_LOCATIONIQ_ACCESS_TOKEN}&q=${city}&format=json`;
+
+    console.log(url);
+    let cityResults = await axios.get(url);
+
+    console.log(cityResults.data[0]);
+    this.setState({
+      cityData: cityResults.data[0]
+    })} catch(error){
+      this.setState({
+        renderError: true,
+        errorMessage: `Error Occured: ${error.response.status}, ${error.response.data.error}`
+
+      })
+    }
+  }
```

Comment:
- Good work here! 👍  We can clean up the indentation for code inside the try block